Cargo.lock

```diff
@@ -2115,7 +2115,7 @@ source = "git+https://github.com/fominok/jsonschema-rs?branch=feat-unevaluated-p
 dependencies = [
  "ahash 0.7.6",
  "anyhow",
- "base64 0.21.2",
+ "base64 0.13.1",
```

Comment:
Why did you downgrade it?

Reply:
it is automatically, once I've run `cargo update`